core/drivers/scmi-msg/common.h

```diff
@@ -15,8 +15,9 @@
 #include "base.h"
 #include "clock.h"
 #include "reset_domain.h"
+#include "voltage_domain.h"
```

Comment:
These header files are not used in `common.h`, are they? In this case I think it would be better not to include them here.

Reply:
No, not strictly needed. Actually the local header files do not strictly reflect related source files.
common.h is used for generic scmi-msg and for smci protocols specific thinks.
Instead of this change, would you rather I make a bigger change to cleanup these #include and the local header files content?

Reply:
Yes please

Reply:
done